src/NServiceBus.PersistenceTests/IPersistenceTestsConfiguration.cs

```diff
@@ -48,13 +48,13 @@ public partial class PersistenceTestsConfiguration : IPersistenceTestsConfigurat
     {
         public PersistenceTestsConfiguration(TestVariant variant)
         {
+            SessionTimeout = variant.SessionTimeout;
```

Comment:
Was there a reason for moving these?

Reply:
yes.  The SupportsDtc check was throwing a null reference because Variants needs to be set.